packages/expo-cli/src/prompts.ts

```diff
@@ -0,0 +1,34 @@
+import program from 'commander';
+import prompts, { Choice, PromptObject, PromptType } from 'prompts';
+
+import CommandError from './CommandError';
+
+export { PromptType, PromptObject };
```

Comment:
```suggestion
export { PromptType, PromptObject as Question };
```
Maybe to make it easier to refactor? Plus the name `PromptObject` doesn't come from the original library (they're called ["questions"](https://github.com/terkelg/prompts/blob/master/lib/index.js#L10) in `prompts` source code), but from `@types/prompts` and the `Object` suffix is redundant.

Reply:
Ah, gotcha, sure I'll export them with the existing names 😄

Reply:
sounds good to me